plugins/tours/tests/phpunit/integration/admin/addDescriptionBeneathPostTitle.php

```diff
@@ -0,0 +1,67 @@
+<?php
+/**
+ * Tests for add_description_beneath_post_title().
+ *
+ * @package     spiralWebDb\CornerstoneTours\Tests\Integration
+ * @since       1.0.0
+ * @author      Robert Gadon <rgadon107>
+ * @link        https://github.com/rgadon107/cornerstone
+ * @license     GNU-2.0+
+ */
+
+namespace spiralWebDb\CornerstoneTours\Tests\Integration;
+
+use spiralWebDb\Cornerstone\Tests\Integration\Test_Case;
+use Brain\Monkey;
+use function spiralWebDb\CornerstoneTours\add_description_beneath_post_title;
+
+/**
+ * Class Tests_AddDescriptionBeneathPostTitle
+ *
+ * @package spiralWebDb\CornerstoneTours\Tests\Integration
+ * @group   tours
+ * @group   admin
+ */
+class Tests_AddDescriptionBeneathPostTitle extends Test_Case {
+
+	/**
+	 * Instance of the post object for each test.
+	 *
+	 * @var WP_Post
+	 */
+	protected $post;
+
+	/**
+	 * Test add_description_beneath_post_title() should return empty when post type is 'post'.
+	 */
+	public function test_should_return_empty_when_post_type_is_post() {
+		// Create and get the post ID for the 'post' post_type via factory method.
+		$post_id = self::factory()->post->create();
+		get_post_type( $post_id );
+
+		$this->assertEmpty( add_description_beneath_post_title() );
+	}
```

Comment:
When doing an integration test for a callback:

1. Don't directly invoke the callback. Why? It's automatically invoked when the event it's hooked to fires. 
2. Find a way to get the event to fire or directly fire the event.

In this case, the callback is registered to the action event named `edit_form_before_permalink`. What in Core makes this event fire? You can automatically have this fire. But in this case, I'm thinking it might be easy enough to fire it in the test. How?

```php
public function test_should_return_empty_when_post_type_is_post() {
	// Create and get the post ID for the 'post' post_type via factory method.
	$post= self::factory()->post->create_and_get();

	ob_start();
	do_action( 'edit_form_before_permalink', $post );
	$actual_html = ob_get_clean();
	
	$view_html = <<<VIEW
<span class="description">Enter the theme name above for this Cornerstone tour. In the editor below, add each of the venues and locations (city, state) where Cornerstone performed on this tour. Below the editor, enter additional tour information in the box labeled "Past Tour Custom Fields".</span>
VIEW;

	$this->assertNotContains( $view_html, $actual_html );
}
```

Reply:
@hellofromtonya Superb advice and example above.  I used this guidance in all the PRs (#58 - 62) that you recently reviewed, and which I am resubmitting after updating.